cardano-constitution/src/Cardano/Constitution/Validator/Dynamic.hs

```diff
@@ -0,0 +1,83 @@
+{-# LANGUAGE LambdaCase        #-}
+{-# LANGUAGE NoImplicitPrelude #-}
+{-# LANGUAGE TemplateHaskell   #-}
+-- Following is for tx compilation
+{-# LANGUAGE DataKinds         #-}
+{-# OPTIONS_GHC -fplugin PlutusTx.Plugin #-}
+{-# OPTIONS_GHC -fplugin-opt PlutusTx.Plugin:target-version=1.1.0 #-}
+module Cardano.Constitution.Validator.Dynamic
```

Comment:
Could you clarify what the static and dynamic versions are for? I'm not really sure what's going on.  Is this to allow for efficiency when we're using the default version of the parameters but also to allow a different version to be supplied at runtime?  We do something like that for the cost model parameters.